### PR TITLE
[author][new] add vega-tooltip w/ npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by sirahd**
Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/vega/vega-tooltip
 * NPM package url (optional): https://www.npmjs.com/package/vega-tooltip
 * License and its reference: BSD-3
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 8
   - Count of stars: 16
   - Count of forks: 4
 * NPM download stats (optional):
   - Downloads in the last day: 16
   - Downloads in the last week: 25
   - Downloads in the last month: 73
 * This is the plugin for vega and vega-lite modules (https://github.com/vega/vega and https://github.com/vega/vega-lite) that provides tooltip to extend the capability.

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by domoritz** [comment]
This addition is coming from the same group that created https://cdnjs.com/libraries/vega, https://cdnjs.com/libraries/vega-lite, and https://cdnjs.com/libraries/vega-embed. Wite vega-tooltip does not have the popularity yet, we hope it will have soon as we are going to release Vega 3 and Vega-Lite 2.

File: vega-tooltip/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "type": "git",
+    "url": "https://github.com/vega/vega-tooltip.git"
+  },
+  "license": "BSD-3-Clause",
+  "npmName": "vega-tooltip",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/vega-tooltip.+(js|css|map)"
```
Comment (pvnr0082t) [inline vega-tooltip/package.json]: @sirahd If there are no other files like `.html` or `.scss`, etc, I think we can use more powerful rule `**/*`. How do you think?

Comment (domoritz) [inline vega-tooltip/package.json]: Same as in https://github.com/cdnjs/cdnjs/pull/10760#discussion_r103609110. We might add the compiled js files for every single ts file to build in the future.

Comment (pvnr0082t) [inline vega-tooltip/package.json]: @domoritz Thanks for the explanation. :+1:

**Comment by pvnr0082t** [comment]
Help rebase and modify the commit message.

**Comment by PeterDaveHello** [comment]
@pvnr0082t @sirahd please note that the path of this lib was wrong, I'll fix it.

**Comment by sirahd** [comment]
@PeterDaveHello Thanks! I appreciate it.

**Pull request merged**